dashboard/client/public/logout.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Redirecting to OpenFaaS</title>
+</head>
+<body>
+    <div>
+        Signed out successfully. Redirecting.. <a href="redirect">to OpenFaaS</a>.
+    </div>
+    <script>
+        window.location.replace("https://openfaas.com");
```

Comment:
Better use `www.openfaas.com` since sometimes the raw domain can have issues with SSL configs.